solr/test-framework/src/java/org/apache/solr/BaseDistributedSearchTestCase.java

```diff
@@ -624,19 +624,18 @@ protected QueryResponse query(boolean setDistribParams, Object[] q) throws Excep
 
   /** Returns the distributed QueryResponse */
   protected QueryResponse query(boolean setDistribParams, SolrParams p) throws Exception {
+    if (p.get("distrib") != null) {
+      throw new IllegalArgumentException("don't pass distrib param");
+    }
 
-    final ModifiableSolrParams params = new ModifiableSolrParams(p);
-
-    // TODO: look into why passing true causes fails
-    params.set("distrib", "false");
```

Comment:
I didn't think an explicit value should have been passed to the control client.  I made this change to help the tests in the other PR work when standalone was being tested, in which distrib has a different default based on `setForcedDistrib`

`true` makes no sense here to me -- I don't get the TODO above it.

But if we don't accept the change in the first rev of this PR that changes the default `distrib` value, then maybe we don't need this either.  This could go in the other PR.